### ISSUE TITLE
NullReferenceException Error

### ISSUE THREAD

**Issue by ACH1980** [dkavolis/Ferram-Aerospace-Research#116]
Hey, this plugin and the Kerbalism plugin caused a NullReferenceException. Is this an incompatibility or is this another bug that should be fixed? Below is the KSP.log of the not starting game.
[KSP.log](https://github.com/dkavolis/Ferram-Aerospace-Research/files/6563514/KSP.log)

**Comment by dkavolis** [comment]
Looking at the log and [`KERBALISM.DB.KerbalismData (Vessel vessel)`](https://github.com/Kerbalism/Kerbalism/blob/6fc4503048dd766d6d29aece9f167214bdfae24f/src/Kerbalism/Database/DB.cs#L185-L195) `PartModule.vessel` seems to be `null` which is controlled by KSP. Are you sure that all your mods and parts are not broken somehow?

**Comment by ACH1980** [comment]
You mean maybe Kerbalism is broken? I could reinstall this and look what's happening then. All Mods reinstalling would take very long...

**Comment by dkavolis** [comment]
Kerbalism should be fine. Your log contains lots of `PartCompiler` errors, try fixing those for a start, and then other errors in the log file.

**Comment by ACH1980** [comment]
Ok, I reinstalled the whole Game and all Mods, but that didn't worked at all (at least the `PartCompiler` errors were triggered later (after selecting a Savegame)). The long pause was caused by my own helplessness and I hoped with some updates, that the files get reinstalled. And that happened, today I tested again and the `PartCompiler` errors where away, but I've other errors that point to corrupted files. I'll try reinstalling them and because of corrupted files are obviously one of the most common problems, I'll request a Feature for CKAN to validate the mods (should be easily done by hashing them) and the option to reinstall on failed validation...
